### PR TITLE
CSS Modules change

### LINKED ISSUE #149: CSS Global Selectors Causing Inconsistent Styles
**Describe the bug**
CSS selectors are defined globally which is causing issues with how some components are being displayed.

**To Reproduce**
1. Go to Journal view
2. Try to add widgets (i.e. habit tracker) to the journal view
3. See that there is some inconsistent style such as content bleeding over borders.

**Expected behavior**
Styles should be displayed properly.

**Screenshots**
CssIssue

**Desktop (please complete the following information):**
 - OS: MacOs Big Sur
 - Browser Chrome

### PR DESCRIPTION
**Pull request by a-158w**
Closes #149

## Proposed Changes

- All css files changed to modules
- All uses also changed in line with the change to modules.
There will be some conflicts due to the changes made in all files the were previously not using css modules. Therefore I recommend not starting any new features until this PR is merged - and being prepared to accept the conflicting css changes on features currently being worked on.

### CODE REVIEW

**Review by ysway — APPROVED** [review]
Thanks for your hard-working and debugging.
Since these conflicts are only related to CSS style issues, I recommend force update the old files.

**Review by Benjamin-Piper — APPROVED** [review]
Let's get this merge on the road!

File: telos-frontend/src/components/text/TextWidget.module.css [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-    height: inherit;
-    background-color: transparent;
-    border: none;
-    outline: none;
+  resize: none;
+  width: inherit;
+  height: inherit;
+  background-color: transparent;
+  border: none;
+  outline: none;
```
Comment (JChamberlain28) [inline telos-frontend/src/components/text/TextWidget.module.css:7]: Were the spaces removed here due to the linter? Should there be indentation?

**Review by JChamberlain28 — CHANGES_REQUESTED** [review]
Please see my comment regarding CSS indentation.

File: telos-frontend/src/components/text/TextWidget.module.css (hunk 1 above)
Comment (a-158w) [reply telos-frontend/src/components/text/TextWidget.module.css:7]: This code already exists on upstream in this form.

Comment (JChamberlain28) [reply telos-frontend/src/components/text/TextWidget.module.css:7]: It is showing that you have removed the spaces

**Review by ambiyu — APPROVED** [review]
LGTM

**Review by JChamberlain28 — APPROVED** [review]

File: telos-frontend/src/components/text/TextWidget.module.css (hunk 1 above)
Comment (JChamberlain28) [reply telos-frontend/src/components/text/TextWidget.module.css:7]: Was informed that prettier made these changes. Commenting here to keep a record. Happy with the explanation :)

**Pull request merged**